tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -149,6 +166,50 @@ func NewStresser(s string, sc *stressConfig, m *member) Stresser {
 			keysPerLease: sc.keysPerLease,
 			rateLimiter:  sc.rateLimiter,
 		}
+	case "election-runner":
```

Comment:
these should provision ops for themselves from `sc.rateLimiter` so they're not invisible to the rate limiting; probably via `SetLimit()`